src/fides/core/deploy.py

```diff
@@ -97,6 +98,39 @@ def check_docker_version() -> bool:
     return version_is_valid
 
 
+def check_virtualenv() -> bool:
+    """
+    Check to see if the deploy is running from a virtual environment, and log a
+    warning if not.
+
+    In many cases, running outside of a virtual environment will lead to
+    mysterious Docker or Python errors, so warning the user helps get ahead of
+    some of these problems.
+    """
+
+    # Adapted from https://stackoverflow.com/a/58026969/
+    # Detect a virtualenv by checking sys.prefix if the  "base" prefixes match
+    real_prefix = getattr(sys, "real_prefix", None)  # set by older virtualenv
+    base_prefix = getattr(sys, "base_prefix", sys.prefix)
+    running_in_virtualenv = (base_prefix or real_prefix) != sys.prefix
+    print(f"in check_virtualenv(), {sys.base_prefix}, {sys.prefix}")
```

Comment:
This looks like a leftover print from debugging.

Reply:
Oh thank you!